test/testdata/circular-is.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    creationTimestamp: 2016-12-02T18:55:30Z
```

Comment:
you should likely get rid of these fields (creationTimestamp, generation, resourceVersion, selfLink, uid)